include/cuco/static_multiset.cuh

```diff
@@ -82,7 +82,7 @@ template <class Key,
           class ProbingScheme      = cuco::double_hashing<4,  // CG size
                                                      cuco::default_hash_function<Key>>,
           class Allocator          = cuco::cuda_allocator<Key>,
-          class Storage            = cuco::storage<1>>
+          class Storage            = cuco::storage<2>>
```

Comment:
Is this a better default?

Reply:
Yes, vector load provided higher performance for multimap/set while singular load works better with single map/set